src/components/shared/cellRenderers/cellRenderer.scss

```diff
@@ -35,6 +35,22 @@ $iconSize: 14px;
     > .icon-only { display: none !important; }
   }
 
+  .soft-select-link-cell {
+    display: inline;
+    color: inherit;
+    text-decoration: underline;
+
+    &:hover { text-decoration: none; }
+  }
+
+  .glimmer-icon {
+    svg {
+      @include square-px-rem(10px);
+      @include rem-fallback(margin-left, -10px);
```

Comment:
May be, if you convert margin-left to padding left, the glimmer icon collapsing with other elements will be fixed.

Margins are considered to be outside of the element, and margins of adjacent items will overlap. The biggest difference between padding and margin is that vertical margins auto-collapse, and padding doesn't. This is why  padding is considered to be part of the element, and is always preserved

Reply:
I can play with it some more, but our goal was not to mess with the grid or its cells as they are today.  And the cells use margin (not padding now).  In order to accommodate the glimmer using padding, we would have to modify the cells to use it... which could impact its ability to auto size the columns.